src/thunderbots/software/grsim_communication/grsim_command_primitive_visitor.cpp

```diff
@@ -153,14 +153,12 @@ void GrsimCommandPrimitiveVisitor::visit(const PivotPrimitive &pivot_primitive)
 
 void GrsimCommandPrimitiveVisitor::visit(const DribblePrimitive &dribble_primitive)
 {
-    // TODO: https://github.com/UBC-Thunderbots/Software/issues/107
-
-    float kickPower = dribble_primitive.isSmallKickAllowed() ? 0.5f : 0.0f;
+    float kickPower =
```

Comment:
nit: if you just make this a `double` you won't need to cast. And if you are going to cast, use `static_cast`. It's slightly safer / better convention.